packages/webapp/src/types.ts

```diff
@@ -134,20 +134,26 @@ export interface Connection {
     oauthToken: string | null;
     oauthTokenSecret: string | null;
     rawCredentials: object;
-    credentials: BasicApiCredentials | ApiKeyCredentials | null;
+    credentials: BasicApiCredentials | ApiKeyCredentials | OAuthOverride | null;
 }
 
 export interface BasicApiCredentials {
-    [key: string]: string;
     username: string;
     password: string;
 }
 
 export interface ApiKeyCredentials {
-    [key: string]: string;
```

Comment:
Isn't this going to break some things? Don't we have weird cases of API key auth where users need to pass in more fields?

Reply:
This is just the webapp types so it won't break anything that a user is doing.